dart/constraint/ConstrainedGroup.cpp

```diff
@@ -108,16 +108,20 @@ void ConstrainedGroup::removeAllConstraints()
 }
 
 //==============================================================================
-bool ConstrainedGroup::containConstraint(ConstraintBase* _constraint) const
+bool ConstrainedGroup::containConstraint(
+    const ConstConstraintBasePtr& _constraint) const
 {
 //  std::cout << "CommunityTEST::_containConstraint(): Not implemented."
 //            << std::endl;
 
+  // TODO(MXG): Is there any reason these functions are not implemented yet?
```

Comment:
No reason. It seems I just forgot to implement. I'll implement them in a separate pull request soon.